ui/vss-extension.json

```diff
@@ -139,6 +139,36 @@
                 "order": 110,
                 "uri": "workItemService.html"
             }
+        },
+        {
+            "id": "sample-work-item-menu",
+            "type": "ms.vss-web.action",
+            "description": "Shows the target properties for menu actions for work items",
+            "targets": [
+                "ms.vss-work-web.work-item-context-menu",
+                "ms.vss-work-web.active-work-item-notifications"
```

Comment:
This has changed completely now, next update will contain two contributions, one for the menu item and one for the listener.  Should be clearer.